repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -4,6 +4,8 @@ include::_includes/in-progress.adoc[]
 
 == Future Release
 
+* https://repose.atlassian.net/browse/REP-5234[REP-5234] - added the new <<services/datastores.adoc#_remote_datastore, Remote Datastore service>> which allows the Distributed Datastore service concept to work in a dynamic containerized environments like OpenShift.
```

Comment:
"... in a dynamic ... environment ..." or "... in dynamic ... environments ...". Mixed plurality here.